pkg/ccr/base/spec.go

```diff
@@ -1466,6 +1466,23 @@ func (s *Spec) DropRollup(destTableName, rollup string) error {
 	return s.Exec(dropRollupSql)
 }
 
+func (s *Spec) ModifyDistributionBucketNum(destTableName string, bucketType string, autoBucket bool, bucketNum int, columnsName string) error {
+	dbName := utils.FormatKeywordName(s.Database)
+	destTableName = utils.FormatKeywordName(destTableName)
+	modifyDistributionBucketNumSql := "ALTER TABLE " + dbName + "." + destTableName + " MODIFY DISTRIBUTION DISTRIBUTED BY " + bucketType
+	if bucketType == "HASH" {
+		modifyDistributionBucketNumSql += fmt.Sprintf("(%s)", columnsName)
+	}
```

Comment:
![image](https://github.com/user-attachments/assets/0f86236b-fee4-4ea5-9758-c6db561564b7)
random distribution is missing